src/patternfly/demos/Page/page-template-header-tools-elements.hbs

```diff
@@ -1,22 +1,22 @@
 {{#> page-header-tools}}
   {{#> page-header-tools-group page-header-tools-group--modifier="pf-m-icons"}}
-    {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Alerts"'}}
-        <i class="fas fa-bell" aria-hidden="true"></i>
+    {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Unread notifications"'}}
+        {{#> notification-badge notification-badge--modifier="pf-m-unread"}}
+          <i class="fas fa-bell" aria-hidden="true"></i>
+        {{/notification-badge}}
     {{/button}}
-    {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Settings"'}}
+    {{#> button button--modifier="pf-m-plain pf-m-hidden pf-m-visible-on-lg" button--attribute='aria-label="Settings"'}}
         <i class="fas fa-cog" aria-hidden="true"></i>
     {{/button}}
+    {{#> dropdown dropdown--modifier="pf-m-hidden-on-lg" id=(concat page--id '-dropdown-kebab-right-aligned-1') dropdown--IsActionMenu="true" dropdown-menu--modifier="pf-m-align-right" dropdown-toggle--modifier="pf-m-mobile pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}
+    {{/dropdown}}
```

Comment:
@mcoker @mattnolting what are your thoughts on moving the dropdown kebab to the icons group?

Reply:
I think that works since we want the kebab grouped menu to display adjacent to the notification bell.